site/zenodo_rdm/legacy/serializers/__init__.py

```diff
@@ -33,6 +33,27 @@ def unwrap_hits(self, data, many, **kwargs):
         return data.get("hits", {}).get("hits", [])
 
 
+class ZenodoListSchema(BaseListSchema):
+    """Legacy top-level array/list schema for JSON Serializer."""
+
+    links = fields.Method("get_links")
```

Comment:
The method as it is should be added in ``BaseListSchema`` in flask-resources because it affects all serializers using the MarshmallowSerializer.

Here we should still override the method being added to ``BaseListSchmea`` because it will generate links like:

- ``https://zenodo.org/api/records?q=&sort=newest&page=1&size=10``

Which in legacy is:
-  ``https://zenodo.org/api/records?q=&sort=mostrecent&page=1&size=10